### ISSUE TITLE
Improvement with DHCP

### ISSUE THREAD

**Issue by Gandulf78** [AdguardTeam/AdGuardHome#4108]
Have a question or an idea? Please search it [on our forum](https://github.com/AdguardTeam/AdGuardHome/discussions) to make sure it was not yet asked. If you cannot find what you had in mind, please [submit it here](https://github.com/AdguardTeam/AdGuardHome/discussions/new).

### Prerequisites

Please answer the following questions for yourself before submitting an issue. **YOU MAY DELETE THE PREREQUISITES SECTION.**

- [X] I am running the latest version
- [X] I checked the documentation and found no answer
- [X] I checked to make sure that this issue has not already been filed

### Problem Description
Improvements to be made for DHCP:
- allow client creation from the DHCP page or alternatively from the Client page with list of MAC address / name of devices. It is currently overcomplicated to create clients with their MAC address, you have to copy MAC address from one page to the other
- default creation of rules "block/allow this address for this client" based on the name of the device (and not the IP address) to deal with dynamic lease (not sure it is currently working actually)

**Comment by bvandevliet** [comment]
I agree! Creating static DHCP leases should be as easy as with Pi-hole. Maybe the MAC addresses could become hyperlinks, and when clicked, it opens the `New static lease` dialog with `MAC` and `Hostname` fields pre-filled.
